src/SkeletalTrapezoidation.cpp

```diff
@@ -1046,33 +1059,40 @@ void SkeletalTrapezoidation::generateTransitionEnds(edge_t& edge, coord_t mid_po
 #ifdef DEBUG
         if (! generateTransitionEnd(edge, start_pos, end_pos, transition_half_length, mid_rest, end_rest, lower_bead_count, edge_transition_ends))
         {
-            RUN_ONCE(logWarning("There must have been at least one direction in which the bead count is increasing enough for the transition to happen!\n"));
+            spdlog::warn("There must have been at least one direction in which the bead count is increasing enough for the transition to happen!"));
```

Comment:
There is a extra `)`.

Reply:
Good  catch!

Fixed in https://github.com/Ultimaker/CuraEngine/commit/9001977cb295bb445aa3dcb420ef7e2ae721204c